docs/development/testmachinery_tests.md

```diff
@@ -326,9 +326,9 @@ Test machinery tests usually deploy a workload to the Shoot cluster as part of t
 - Do not use container images from Docker Hub.
   - Docker Hub has rate limiting (see [Download rate limit](https://docs.docker.com/docker-hub/download-rate-limit/)). For anonymous users, the rate limit is set to 100 pulls per 6 hours per IP address. In some fenced environments the network setup can be such that all egress connections are issued from single IP (or set of IPs). In such scenarios the allowed rate limit can be exhausted too fast. See https://github.com/gardener/gardener/issues/4160.
   - Docker Hub registry doesn't support pulling images over IPv6 (see [Beta IPv6 Support on Docker Hub Registry](https://www.docker.com/blog/beta-ipv6-support-on-docker-hub-registry/)).
-  - Avoid manually copying Docker Hub images to Gardener GCR (`eu.gcr.io/gardener-project/3rd/`). Use the existing prow job for this (see [Copy Images](https://github.com/gardener/ci-infra/tree/master/config/images)).
+  - Avoid manually copying Docker Hub images to Gardener GCR (`europe-docker.pkg.dev/gardener-project/public/3rd/`). Use the existing prow job for this (see [Copy Images](https://github.com/gardener/ci-infra/tree/master/config/images)).
   - If possible, use a Kubernetes e2e image (`registry.k8s.io/e2e-test-images/<image-name>`).
     - In some cases, there is already a Kubernetes e2e image alternative of the Docker Hub image.
-      - For example, use `registry.k8s.io/e2e-test-images/busybox` instead of `eu.gcr.io/gardener-project/3rd/busybox` or `docker.io/busybox`.
+      - For example, use `registry.k8s.io/e2e-test-images/busybox` instead of `europe-docker.pkg.dev/gardener-project/public/3rd/busybox` or `docker.io/busybox`.
```

Comment:
```suggestion
      - For example, use `registry.k8s.io/e2e-test-images/busybox` instead of `europe-docker.pkg.dev/gardener-project/releases/3rd/busybox` or `docker.io/busybox`.
```